### DISCUSSION TITLE
I made some changes to 3.02 pptx. Available for comment.

### DISCUSSION THREAD

**Discussion by jdonwells** [TEALSK12/2nd-semester-introduction-to-computer-science#350] [Ideas]
The first change was to use parameter and argument as per the college board. 

I then changed function contract in comments to a docstring.

I then changed the abstraction to something more specific. Most students will not know about toe out. So I specifically called that out as the steering wheel abstraction.

https://docs.google.com/presentation/d/1Z_Aiyc9gI0QTH8SzjlPvjlSD3DTOezsV5AQrJBExC70/edit#slide=id.p1

**Comment by tmorris1234** [comment]
Love the way you changed the abstraction piece.  My only issue with the presentation, and it is a small one, is that I hadn't introduced try / except syntax (or the concept in general of error catching).

**Reply by jdonwells** [reply]
That is intentional. Tell them they can take that code and type it in and it would work for them. They don't need to know how it works because it is an abstraction. It could even be a library.

CS joke recommendation: Point to try: and say "try? There is only do!"
